pkg/workflow/controller/trigger/main.go

```diff
@@ -0,0 +1,104 @@
+package trigger
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	api_v1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/workqueue"
+)
+
+func getKubernetesClient() kubernetes.Interface {
```

Comment:
Create an util package to new k8s client, as it is used in several places. @cd1989  @zhujian7

Reply:
I think so.
I meet some problem without `util package to new k8s client`, and I create a tmp one in some place to use.